README.md

```diff
@@ -113,138 +86,195 @@ You can still find the latest installers in the [24.9.2 release](https://github.
 
 </details>
 
-## Install
 
-### Unix-like platforms (macOS & Linux)
+## Usage
 
-Download the installer using curl or wget or your favorite program and run the script.
-For eg:
+If Miniforge is on the system path (default on macOS and Linux), its versions of the
```

Comment:
I want to change the whole idea of "Miniforge" being on the path.

"Miniforge" gets "conda" and "mamba" on your system. Once it is installed, its "job" is done. I want to help emphasize this.